app/components/Wallet/ReceiveModal/ReceiveModal.js

```diff
@@ -58,116 +80,99 @@ class ReceiveModal extends React.Component {
       return null
     }
 
-    let qrCode
+    let qrCode, message
     switch (qrCodeType) {
```

Comment:
Would be nice to have `default` here too

Reply:
Have added a default message for this one